### ISSUE TITLE
[graphql-ruby 1.10] Resolve error when extending a schema class

### ISSUE THREAD

**Issue by jonian** [DmitryTsepelev/graphql-ruby-persisted_queries#31]
Hi, thank you for this gem.

I'm getting errors from [resolver](https://github.com/DmitryTsepelev/graphql-ruby-persisted_queries/blob/master/lib/graphql/persisted_queries/resolver.rb) when extending schema classes and using this gem. 

Example setup:
```ruby
class ApplicationSchema < GraphQL::Schema
  use ::GraphQL::PersistedQueries,
    store: :redis,
    redis_client: Redis.current
end

class MySchema < ApplicationSchema
end
```
Error log:
```
NoMethodError (undefined method `call' for nil:NilClass):
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/resolver.rb:44:in `rescue in with_error_handling'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/resolver.rb:41:in `with_error_handling'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/resolver.rb:32:in `resolve'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/multiplex_resolver.rb:35:in `resolve_persisted_query'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/multiplex_resolver.rb:27:in `block in resolve_persisted_queries'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/multiplex_resolver.rb:26:in `each'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/multiplex_resolver.rb:26:in `each_with_index'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/multiplex_resolver.rb:26:in `resolve_persisted_queries'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/multiplex_resolver.rb:14:in `resolve'
graphql-persisted_queries (0.5.0) lib/graphql/persisted_queries/schema_patch.rb:52:in `multiplex'
graphql (1.10.5) lib/graphql/schema.rb:1546:in `execute'
```
It looks that any patches applied in [SchemaPatch](https://github.com/DmitryTsepelev/graphql-ruby-persisted_queries/blob/master/lib/graphql/persisted_queries/schema_patch.rb) are not passed to extended classes and `@schema.persisted_query_store`, `@schema.persisted_query_error_handler` return `nil`.

**Comment by DmitryTsepelev** [comment]
Hi @jonian!

Interesting, I guess no one tried to use it with the extended classes (not related to the issue, but I wonder the reason behind inheriting schemas inside the application), I'll try to take a look later this week. At the meantime, copying plugin initialization too to `MySchema` will probably help for now

**Comment by jonian** [comment]
Thank you for the quick response @DmitryTsepelev! 

I fixed the issue, as you suggest, by adding the plugin initialization to `MySchema` and removing it from `ApplicationSchema` (has to be removed to work, stack level too deep errors).

> I wonder the reason behind inheriting schemas inside the application

That was an example, I'm not inheriting the schema inside the application. I have a gem that I use across many applications/micro-services and it provides a schema class with error-handler, persisted-queries, tracing etc. enabled.

While I can achieve the same results by including mixins or methods like `enable_persisted_queries` on each schema class, it would be nice if inheritance worked.

**Comment by DmitryTsepelev** [comment]
Yeah, I guess I'll need to hook the inheritance somehow and copy storage settings

**Comment by DmitryTsepelev** [comment]
Fixed in #43
